signalfx_lambda/utils.py

```diff
@@ -65,6 +65,9 @@ def get_tracing_url():
         url = os.environ.get('SIGNALFX_ENDPOINT_URL')
 
         if url:
+            # if the common endpoint url is used with a trailing slash, we remove the trailing slash
```

Comment:
You'll end up with cleaner code by using `urlparse.urljoin()`. It will handle the trailing slash for you. This method can be rewritten like so:

```python
import six.moves.urllib.parse as urlparse

def get_tracing_url():
  url = os.environ.get('SIGNALFX_TRACING_URL')
  if url:
    return url

  url = os.environ.get('SIGNALFX_ENDPOINT_URL', 'https://ingest.signalfx.com')
  return urlparse.urljoin(url, '/v1/trace')
```

Nicer, isn't it? :)

Reply:
Most definitely. Thanks!